src/test/java/ArrayMinHeapTest.java

```diff
@@ -25,19 +25,113 @@ void size_empty_returnsZero() {
                     .isEqualTo(0);
         }
 
+        @Test
         void isEmpty_empty_returnsTrue() {
             assertThat(classUnderTest.isEmpty())
                     .isTrue();
         }
 
+        @Test
+        void add_empty_returnsTrue() {
+            assertThat(classUnderTest.add(10))
+                    .isTrue();
+        }
```

Comment:
[Is this what you meant?](https://github.com/jameshughes89/cs201/pull/13#discussion_r1021976600)

Reply:
1